.github/workflows/build.yml

```diff
@@ -0,0 +1,80 @@
+name: Build
+
+on:
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+      - published
+  push:
+    branches:
+      - master
+
+env:
+  kernel: "6.6.36"
```

Comment:
now 6.6.38 is available, please raise to 6.6.37 that we have one shoot left to after the merge